src/sentry/static/sentry/app/components/events/interfaces/utils.jsx

```diff
@@ -1,5 +1,6 @@
 import {isString} from 'lodash';
 import * as Sentry from '@sentry/browser';
+import queryString from 'query-string';
```

Comment:
I think we generally import this as `qs?` but *shrug* that's default imports for ya.